patterns/2-structured/start-as-experiment.md

```diff
@@ -4,94 +4,52 @@ Start as an Experiment
 
 ## Forces
 
-- Managers will want to validate the claims of improved collaboration through
-  InnerSource before making a long term investment. This usually involves
-  measuring the improvements.
-- If the InnerSource initiative will likely have a huge uptake among developers
-  and if many projects are likely to rely on it, a decision to shut it down
-  will be very unpopular and therefore hard to make. The perceived resulting
-  loss of control might deter some managers to even start with InnerSource.
-- Implementing InnerSource style working models are often a radical departure
-  from previously practiced working models. It is therefore likely, that
-  existing, mandatory processes are no longer applicable and that appropriate
-  governing processes are missing. The result might be that one has to operate
-  in a regulatory, sometimes legal no-mans land. Examples are tax and export
-  control related regulations in large corporations with multiple legal
-  entities in multiple countries.
+- Managers will want to validate the claims of improved collaboration through InnerSource before making a long term investment. This usually involves measuring the improvements.
+- If the InnerSource initiative will likely have a huge uptake among developers and if many projects are likely to rely on it, a decision to shut it down will be very unpopular and therefore hard to make. The perceived resulting loss of control might deter some managers to even start with InnerSource.
+- Implementing InnerSource style working models are often a radical departure from previously practiced working models. It is therefore likely, that existing, mandatory processes are no longer applicable and that appropriate governing processes are missing. The result might be that one has to operate in a regulatory, sometimes legal no-mans land. Examples are tax and export control related regulations in large corporations with multiple legal entities in multiple countries.
 
 ## Solution
 
-Declare the InnerSource initiative as a time limited experiment. Define and
-communicate the criteria for projects to join the InnerSource experiment. Choose
-criteria that will maximize the chances of building a healthy A set of criteria is
-a good one if the insights generated from it within the context of the experiment
-can intuitively be applied to contexts involving other potential InnerSource projects.
+Declare the InnerSource initiative as a time limited experiment. Define and communicate the criteria for projects to join the InnerSource experiment. Choose criteria that will maximize the chances of building a healthy community. A set of criteria is a good one if the insights generated from it within the context of the experiment can intuitively be applied to contexts involving other potential InnerSource projects.
```

Comment:
@gruetter here I actually made a change to the content. Unfortunately hard to see because I also changed the line-breaks. Sorry! 

Previously the text was:
> Choose criteria that will maximize the chances of **building a healthy A set of criteria** is a good one if the insights generated from it ...

Thee seemed to be something missing in the middle. So I fixed it to be:
> Choose criteria that will maximize the chances of **building a healthy community. A set of criteria** is a good one if the insights generated from it ...

Would you mind checking if that captures the intended meaning?

Reply:
It does. I have no clue where all the formatting and copy&paste errors are coming from, by the way.